src/main/java/io/axoniq/axonserver/connector/impl/AbstractBufferedStream.java

```diff
@@ -124,7 +123,7 @@ public boolean isClosed() {
     public void close() {
         if (!clientClosed.getAndSet(true)) {
             super.close();
-            doIfNotNull(outboundStream(), StreamObserver::onCompleted);
+            doIfNotNull(outboundStream(), s -> s.cancel("Cancelling Stream", null));
```

Comment:
The outboundStream is already automatically closed because Grpc automatically completes the stream. One of the test cases has revealed the problem and we fixed it in this PR.